ag/enrollment.go

```diff
@@ -5,6 +5,12 @@ import (
 	"time"
 )
 
+//gracePeriod is the grace-Period for deliveries after the deadline. It can only be <int> hours.
```

Comment:
Please use space after `//` comments; using the `gofumpt` formatter should fix this when you save.
Please write "grace period" without the hyphen.
Please combine the limitations into one sentence, e.g. "The grace period must be in the range 0-23 hours."
You can remove the "current 2 hour" comment. It is clear from the code.